client/components/gpu-listings/GpuCard.module.css

```diff
@@ -0,0 +1,44 @@
+.wrapper {
+  background-color: #ffffff;
+  overflow: hidden;
+  padding: 2rem;
+  box-shadow: var(--shadow);
+  min-width: 250px;
+  width: 250px;
+}
+
+.divider {
+  color: var(--off-white);
+  opacity: 30%;
+  margin: 1rem 0;
+}
+
+.brand {
+  text-transform: uppercase;
+  font-size: 01rem;
```

Comment:
Should this be 0.1rem or just 1rem?

Reply:
This should be 1, I must of missed the 0 lol